### PR TITLE
In hiding Page

### PR DESCRIPTION
**Pull request by matt-lethargic**

### CODE REVIEW

File: cypress/integration/InHidingPage.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Feature: Are you or your relatives in hiding or in danger page
+  As a user
+  I want to see options on my hiding or danger status
```
Comment (WillGibson) [inline cypress/integration/InHidingPage.feature]:
```suggestion
  I want to see options based on my relatives hiding or danger status
```

File: cypress/integration/InHidingPage.feature [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    When I click the "back" link
+    Then I am on the "where should we start looking" page
+
+  Scenario: Selecting No Im not worried about searching in public and clicking continue navigates to the Eligible for Both Services page
+    When I select "no Im not worried about searching in public" radio button
+    Then I click the "continue" button
+    Then I am on the "eligible for both" page
+
+  Scenario: Selecting No Im not worried about searching in public and clicking continue navigates to the Eligible for Both Services page
+    When I select "yes publicily search for my family could be dangerous" radio button
```
Comment (stevehenshaw) [inline cypress/integration/InHidingPage.feature]: minor typo in 'publicly'

**Review by stevehenshaw — APPROVED** [review]

File: src/views/in-hiding.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      </div>
+
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+
+          {{ govukBackLink({
+                    text: "Back",
+                    href: "/where-start-looking",
+                    classes: "redcross-back-link"
+                }) }}
```
Comment (WillGibson) [inline src/views/in-hiding.html:27]: Indenting...
```suggestion
          {{ govukBackLink({
            text: "Back",
            href: "/where-start-looking",
            classes: "redcross-back-link"
          }) }}
```

File: cypress/integration/InHidingPage.feature [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    Then the progression bar is at "check eligibility"
+
+  Scenario: Has a back link to navigate to the where to start looking page
+    When I click the "back" link
+    Then I am on the "where should we start looking" page
+
+  Scenario: Selecting No Im not worried about searching in public and clicking continue navigates to the Eligible for Both Services page
+    When I select "no Im not worried about searching in public" radio button
+    Then I click the "continue" button
+    Then I am on the "eligible for both" page
```
Comment (WillGibson) [inline cypress/integration/InHidingPage.feature]: I think quotes around the bit their selecting helps differentiate it as a thing from the rest of the words, also, I think it looks weird without the apostrophe, is there a reason for that?

```suggestion
  Scenario: Selecting "No I'm not worried" about searching in public and clicking continue navigates to the Eligible for Both Services page
    When I select "no i'm not worried about searching in public" radio button
    Then I click the "continue" button
    Then I am on the "eligible for both" page
```

Same applies in rest of this file. Happy to jump on a call and thrash this out if that's quickest.

File: cypress/integration/InHidingPage.feature [hunk 5]
```diff
@@ -1,6 +1,6 @@
 Feature: Are you or your relatives in hiding or in danger page
   As a user
-  I want to see options on my hiding or danger status
+  I want to see options based on my relatives hiding or danger status
```
Comment (WillGibson) [inline cypress/integration/InHidingPage.feature]: Sorry...
```suggestion
  I want to see options based on my relative's hiding or danger status
```

**Review by WillGibson — APPROVED** [review]
Just this suggestion to apply https://github.com/madetech/redcross-family-tracing/pull/18/files#r468475101, but it's not a deal breaker so approving.

[EDIT] Subject to passing pipeline!

**Pull request merged**